common/static/karma_common.conf.js

```diff
@@ -38,7 +38,7 @@ var libraryFiles = [
     {pattern: 'js/vendor/jquery.truncate.js', included: true},
     {pattern: 'js/vendor/mustache.js', included: true},
     {pattern: 'common/js/vendor/underscore.js', included: true},
-    {pattern: 'js/vendor/underscore.string.min.js', included: true},
```

Comment:
It may be that there are actually no tests that depend upon underscore.string. In CMS there are RequireJS dependencies, but in the LMS and common I think they are shimmed so their absence won't matter.